Sources/Puree/Output/BufferedOutput.swift

```diff
@@ -160,8 +160,14 @@ open class BufferedOutput: InstantiatableOutput {
         let newBuffer = Set(buffer.dropFirst(logCount))
         let dropped = buffer.subtracting(newBuffer)
         buffer = newBuffer
-        let chunk = Chunk(logs: dropped)
-        callWriteChunk(chunk)
+        prepareChunk(logs: dropped)
+            .forEach { (chunk) in
+                callWriteChunk(chunk)
+            }
+    }
+
+    open func prepareChunk(logs: Set<LogEntry>) -> [Chunk] {
+        [ Chunk(logs: logs) ]
```

Comment:
This project doesn't require the latest Swift version yet. So some compilers can't recognize this implicit return.